pkgs/development/python-modules/micawber/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, beautifulsoup4 }:
+
+buildPythonPackage rec {
+  pname = "micawber";
+  version = "0.3.4";
+
+  src = fetchFromGitHub {
+    owner  = "coleifer";
+    repo   = "micawber";
+    rev    = "4754f0088f452c5227f808321a2abc38745dc248";
+    sha256 = "0likd1sdpwjmmww3ngaakslb09bcb7sjqjq1s9300gyh7hwgilyy";
+  };
+
+  propagatedBuildInputs = [ beautifulsoup4 ];
+
+  meta = with stdenv.lib; {
+    homepage = http://micawber.readthedocs.io/en/latest/;
+    description = "A small library for extracting rich content from urls";
+    license = licenses.mit;
+    longDescription = ''
+      micawber supplies a few methods for retrieving rich metadata
+      about a variety of links, such as links to youtube videos.
+      micawber also provides functions for parsing blocks of text and html
+      and replacing links to videos with rich embedded content.
```

Comment:
Even though I personally dislike not capitalizing the beginning of a sentence, I see that it's just copied from readthedocs. So I guess just leave it like that :)

Reply:
From the guidelines:

> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.

Reply:
That's for the `description`, not for `longDescription`.